### PR TITLE
Quick fix for Hyatt Parsing Bug

### PR DESCRIPTION
**Pull request by fishcharlie**
This PR implements the quick fix for parsing Hyatt hotels mentioned in #1136.

### CODE REVIEW

**Comment by thismakessand** [comment]
@fishcharlie  Good catch - thanks!  What about something like 

```data = json.loads(response.xpath('//script[contains(text(), "streetAddress")]/text()').extract_first())[0]```

so it isn't dependent on order?

File: locations/spiders/hyatt.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def start_requests(self):
             yield scrapy.Request(url=self.base_url.format(region=region))
 
     def parse_hotel(self, response):
-        data = json.loads(response.xpath('//script[@type="application/ld+json"]/text()').extract_first())[0]
+        data = json.loads(response.xpath('//script[@type="application/ld+json"]/text()').getall().last)[0]
```
Comment (fishcharlie) [inline locations/spiders/hyatt.py]:
```suggestion
        data = json.loads(response.xpath('//script[contains(text(), "streetAddress")]/text()').extract_first())[0]
```

**Comment by fishcharlie** [comment]
@thismakessand See what you think of b98f90d. Great idea! Haven't worked with Ruby in a LONG time haha.

**Comment by thismakessand** [comment]
@iandees  Looks good to me as well, thanks @fishcharlie !

**Pull request merged**